numpy/random/tests/test_random.py

```diff
@@ -352,40 +356,41 @@ def test_bytes(self):
         np.testing.assert_equal(actual, desired)
 
     def test_shuffle(self):
```

Comment:
Oh, I see.  I'll switch this out then.
I guess nose should try to stitch in a traceback to the still running generator (`gen.gi_frame` anyone?).  Let's open a bug there...